enterprise/server/remote_execution/snaploader/snaploader.go

```diff
@@ -839,10 +842,10 @@ func (l *FileCacheLoader) unpackCOW(ctx context.Context, file *fcpb.ChunkedFile,
 func (l *FileCacheLoader) cacheCOW(ctx context.Context, name string, remoteInstanceName string, cow *copy_on_write.COWStore, cacheOpts *CacheSnapshotOptions) (*repb.Digest, error) {
 	ctx, span := tracing.StartSpan(ctx)
 	defer span.End()
-	var dirtyBytes, dirtyChunkCount, emptyBytes, emptyChunkCount int64
+	var dirtyBytes, dirtyChunkCount, emptyBytes, emptyChunkCount, compressedBytesWritten int64
```

Comment:
is this change meant for https://github.com/buildbuddy-io/buildbuddy/pull/8404 ?